modules/manage/pages/manage-xdcr/filter-xdcr-replication.adoc

```diff
@@ -15,6 +15,8 @@ The expression can be applied to:
 * The document's extended attributes
 
 This page explains the practical steps whereby filtering can be performed.
+Note that when entered by means of the _UI_, the expression can be a maximum of 250 bytes (characters) in length: this restriction does _not_ apply to expressions entered by means of the _CLI_ or the _REST API_.
```

Comment:
Could CLI and REST API be made links here (as they are in the next section - `Examples on This Page`)?

Reply:
ok

Reply:
Have added.